gitlab/resource_gitlab_service_jira.go

```diff
@@ -0,0 +1,178 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServiceJira() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServiceJiraCreate,
+		Read:   resourceGitlabServiceJiraRead,
+		Update: resourceGitlabServiceJiraUpdate,
+		Delete: resourceGitlabServiceJiraDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabServiceJiraImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"title": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"updated_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"active": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"url": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)`), "value must be a url"),
```

Comment:
I would think that there is a standard validation for this in terraform?

Reply:
maybe a func with URL.Parse

Reply:
Done!